src/ontology/mondo-edit.obo

```diff
@@ -321689,6 +321689,7 @@ synonym: "EIEE14" EXACT ABBREVIATION []
 synonym: "epileptic encephalopathy, early infantile, 14" EXACT [MONDO:Lexical, OMIM:614959]
 synonym: "epileptic encephalopathy, early infantile, type 14" EXACT [MONDORULE:2]
 synonym: "KCNT1 early infantile epileptic encephalopathy" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "KCNT1-related epilepsy" BROAD [https://www.chop.edu/conditions-diseases/kcnt1-related-epilepsy#\:~\:text=Children%20with%20KCNT1%2Drelated%20developmental%20and%20epileptic%20encephalopathy%20have%20delays\,Poor%20muscle%20tone%20(hypotonia), PMID:30234941]
```

Comment:
@nicolevasilevsky is the web site link needed as an xref for the synonym? The PMID seems sufficient. If the web site link is needed, are you ok if it is trimmed down to remove the text highlighting so https://www.chop.edu/conditions-diseases/kcnt1-related-epilepsy?

Reply:
good idea - I shortened the URL